### PR TITLE
✨ JS-608: Return Elements Using CSS Selectors

### PR DESCRIPTION
**Pull request by ct-aa**
# Related Tickets
- [JS-608](https://codeandtheory.atlassian.net/browse/JS-608)

# Description
Added the **Return Elements Using CSS Selectors** challenge solutions.

# Validation
* Open the **Return Elements Using CSS Selectors** folder.
* Open the app.js.
* The variable navigationLinks should store all the a tags under the nav.
* The variable galleryLinks should store all the a tags under the element with the id(gallery).
* The variable footerImages should store all the img under the footer element.

# Risks
N/A

### CODE REVIEW

**Review by lycoangelo — CHANGES_REQUESTED** [review]
Hi @ct-aa, all the selectors don't match anything on the DOM.

**Comment by ct-aa** [comment]
> Hi @ct-aa, all the selectors don't match anything on the DOM.

Thanks, @lycoangelo, this is now updated.

File: Return Elements Using CSS Selectors/app.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+let navigationLinks = document.querySelectorAll('nav a');
+let galleryLinks = document.querySelectorAll('#gallery a');
+let footerImages = document.querySelectorAll('footer img');
```
Comment (lycoangelo) [inline Return Elements Using CSS Selectors/app.js]: Please use `const` since the value was not reassigned.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: Return Elements Using CSS Selectors/app.js (hunk 1 above)
Comment (ct-aa) [reply Return Elements Using CSS Selectors/app.js]: Thanks, @lycoangelo, this is now updated.

File: Return Elements Using CSS Selectors/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            src="img/twitter-wrap.png"
+            alt="Twitter Logo"
+            class="social-icon"
+        /></a>
+        <a href="https://www.facebook.com/TeamTreehouse"
+          ><img
+            src="img/facebook-wrap.png"
+            alt="Facebook Logo"
+            class="social-icon"
+        /></a>
```
Comment (lycoangelo) [inline Return Elements Using CSS Selectors/index.html]: Please format codes properly.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: Return Elements Using CSS Selectors/index.html (hunk 2 above)
Comment (ct-aa) [reply Return Elements Using CSS Selectors/index.html]: Thanks, @lycoangelo, This is now done.

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**